art/utils.py

```diff
@@ -468,6 +468,26 @@ def float_to_categorical(labels, nb_classes=None):
     return categorical
 
 
+def floats_to_one_hot(labels):
```

Comment:
Could you please add typing and remove the `:type ... ` lines from the docstring?

Reply:
Done.